docs/en/source/followRelationship.rst

```diff
@@ -0,0 +1,329 @@
+*********************************
+Follow Relationships in a Query
+*********************************
+
+Follw Relationships in a Query is a feature that aims to allow us to retrieve entities with their linked sub-entities' data using the join parameter.
+The join parameter has two supported values: *inline* and *flat*
+	- If join=inline, the entity and sub-entities should be represented in a hierarchical structure.
+	- If join=flat, the entity and sub-entities should be represented as they are currently, but target entities will be returned in the result list.
+
+For example:  a person holds several pieces of land type:AgriParcel. Each AgriParcel has some of its own attributes, e.g., soilType, cropType. Each AgriParcel also has its own related weather station (i.e., Relationship to an entity type:WeatherObserved) displaying values holding to current observed weather conditions. We also require a weather forecast (i.e. Relationship to an entity type:WeatherForecast) which covers a wider area encompassing multiple AgriParcels for a predicted weather conditions.
+
+.. figure:: figures/followRelationshipDiagram.png
+
+**Note:** The join parameter is compatible with the option parameter in all scenarios including - normalized and concise.
+
+Example for Follow Relationship in a Query
+---------------------------------------------
+
+1. Create Operation
+=========================
+In order to create the entities, we can hit the endpoint **http://<IP Address>:<port>/ngsi-ld/v1/entities/**  with the given payloads.
```

Comment:
Remove this line "we can hit the endpoint **http://<IP Address>:<port>/ngsi-ld/v1/entities/**"

Reply:
Done